bundles/org.openhab.binding.hive/src/main/resources/ESH-INF/thing/bridge.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="hive"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+	<bridge-type id="bridge">
+		<label>Hive Home Account</label>
+		<description>Provides Access To Your Hive Home Account</description>
+		<config-description>
+			<parameter name="USRNM" type="text" required="true">
```

Comment:
If not too many users are yet using this binding I would prefer to have more standard names for this, like "username"